test/run-e2e.sh

```diff
@@ -209,6 +213,10 @@ parse_args() {
 			RUN_REGEX=$1
 			shift
 			;;
+		--ocp)
```

Comment:
I would prefer to have the tests determine whether they run on OCP or not. I think checking for the presence of the `ClusterVersion` or `ClusterOperator` CRD  should be good enough. We can get this up once during test setup and the `framework` can pass it to all tests. Then a test can `t.Skip` if it can't run.